vyxal/elements.py

```diff
@@ -4869,7 +4892,30 @@ def strict_less_than(lhs, rhs, ctx):
 def strip(lhs, rhs, ctx):
     """Element P
     (any, any) -> a.strip(b)
+    (any, fun) -> all elements in a where the result of b(x) is lowest
     """
+    if vy_type(lhs) == types.FunctionType:
+        lhs, rhs = rhs, lhs
+    if vy_type(rhs) == types.FunctionType:
+        lhs = iterable(lhs, ctx=ctx)
+        if not lhs:
+            return lhs
+        m = None
+        fun_vals = []
+        for item in lhs:
```

Comment:
Won't finish on infinite lists

Reply:
Again, it's not possible to write it in a way that it'll finish on infinite lists